base/setup/lib/utils/partlist.c

```diff
@@ -564,8 +546,8 @@ IsSuperFloppy(
 
 
 /*
- * Inserts the disk region represented by PartEntry into either the primary
- * or the logical partition list of the given disk.
+ * Inserts the disk region represented by PartEntry into either the
+ * primary or the logical partition list of the given disk.
```

Comment:
This is an entirely superfluos change. The old style did not even exceed 80 characters. I do recommend to revert that change altogether.
```suggestion
 * Inserts the disk region represented by PartEntry into either the primary
 * or the logical partition list of the given disk.
```

Reply:
So you're telling me, that I cannot reformat my own comments I wrote in the past? What a joke.